lib/question.sql

```diff
@@ -43,6 +44,7 @@ FROM
         ORDER BY id DESC
         LIMIT 1
     ) AS gj ON TRUE
+    JOIN users u ON (s.auth_user_id = u.user_id)
```

Comment:
Could we rebase this on top of #3518 so that we can use `submissions.user_id` rather than `submissions.auth_user_id`? It might be easiest to wait until #3518 is merged to do this.

Reply:
I'll fix your other review note- do you still want to wait until #3518 is merged to consider this?